DependencyInjection/Compiler/MonologHandlerPass.php

```diff
@@ -10,16 +10,12 @@ class MonologHandlerPass implements CompilerPassInterface
 {
     public function process(ContainerBuilder $container)
     {
-        if (!$container->hasParameter('ekino.new_relic.log_logs') || !$container->hasDefinition('monolog.logger')) {
+        if (!$container->hasParameter('ekino.new_relic.monolog.channels') || !$container->hasDefinition('monolog.logger')) {
```

Comment:
shouldn't we add a test to assert that, when config is disabled, the key channel does not exists?